doc/files/opensc.conf.5.xml.in

```diff
@@ -1141,6 +1141,28 @@ app <replaceable>application</replaceable> {
 							address.
 					</para></listitem>
 				</varlistentry>
+				<varlistentry>
+					<term>
+						<option>cache_private_data = <replaceable>bool</replaceable>;</option>
+					</term>
+					<listitem><para>
+							Whether to cache private card's files (e.g.
+							private keys) on disk in
```

Comment:
Do we really have some private keys that can be cached? I think the private can be considered mostly certificates or some data objects, but private keys are usually not readable at all.